### PR TITLE
CTB: Replace ToggleInput with Checkbox for the d&p switch

### PR DESCRIPTION
**Pull request by gu-stav**
### What does it do?

Replaces the `ToggleInput` with a `Checkbox` for the D&P switch in the CTB.

TODO:

- [x] Update docs https://docs.strapi.io/user-docs/latest/content-types-builder/creating-new-content-type.html#creating-a-new-content-type

### Why is it needed?

We will be adding another option soon and want these input fields to be consistent.

| Before | After |
|-|-|
| Screenshot 2023-01-12 at 19 25 47 | Screenshot 2023-01-23 at 12 41 14 |

### How to test it?

1. Got the CTB and edit a content-type
2. Navigate to advanced settings and validate the base form (D&P) and the injected fields (i18n) are being rendered.

### CODE REVIEW

**Comment by joshuaellis** [comment]
The label isn't very good imo, it could be "Enable draft & publish"?

**Comment by gu-stav** [comment]
@joshuaellis This is in draft for now :) I've asked @maevalienard if she would be up for a small rewording and I suggested:

- Draft & publish: Allows writing draft versions of entries before they are published
- Localization: Allows translating entries into different languages

Wdyt?

**Comment by joshuaellis** [comment]
Im not a fan of enable either so works for me.

**Comment by gu-stav** [comment]
After quick discussion and feedback from @maevalienard I'd propose to settle on:


- Draft & publish: Allows writing a draft version of an entry, before it is published
- Localization: Allows translating an entry into different languages

Does that work for you @maevalienard and @joshuaellis ?

**Comment by maevalienard** [comment]
Let's go!

File: packages/core/content-type-builder/admin/src/components/DraftAndPublishToggle/index.js [hunk 1]
```diff
@@ -66,65 +56,28 @@ const DraftAndPublishToggle = ({
...
+
+      <ConfirmDialog
+        isOpen={showWarning}
+        onToggleDialog={handleToggle}
+        onConfirm={handleConfirm}
+        bodyText={{
+          id: getTrad('popUpWarning.draft-publish.message'),
+          defaultMessage:
+            'If you disable the draft & publish, your drafts will be deleted. Are you sure you want to disable it?',
+        }}
```
Comment (jhoward1994) [inline packages/core/content-type-builder/admin/src/components/DraftAndPublishToggle/index.js:71]: Modified to match the translation string and made a tiny wording change. This makes more sense to me.
```suggestion
        bodyText={{
          id: getTrad('popUpWarning.draft-publish.message'),
          defaultMessage: 'If you disable draft & publish, your drafts will be deleted.',
        }}
```

**Review by jhoward1994 — CHANGES_REQUESTED** [review]
Looks good, I just have one small comment

**Review by joshuaellis — DISMISSED** [review]

File: packages/core/content-type-builder/admin/src/components/DraftAndPublishToggle/index.js (hunk 1 above)
Comment (gu-stav) [reply packages/core/content-type-builder/admin/src/components/DraftAndPublishToggle/index.js]: Done in 35bbe83248ddbef66864e04f6ec12ef498ffb022

**Comment by maevalienard** [comment]
@gu-stav I just remembered something. The label should be "Internationalization" not "Localization" because this is not the same feature (https://www.techtarget.com/searchcio/definition/localization) :)

**Comment by gu-stav** [comment]
@maevalienard Thanks. Updated in 540046c1ac0712a69019d59354fe1f0eac04571f.

File: packages/plugins/i18n/admin/src/index.js [hunk 2]
```diff
@@ -102,12 +102,12 @@ export default {
                 name: 'pluginOptions.i18n.localized',
                 description: {
                   id: getTrad('plugin.schema.i18n.localized.description-content-type'),
-                  defaultMessage: 'Allow you to have content in different locales',
+                  defaultMessage: 'Allows translating an entry into different languages',
```
Comment (jhoward1994) [inline packages/plugins/i18n/admin/src/index.js:105]: Apologies if this is too nit picky, but this translation string would make more sense to me
```suggestion
                  defaultMessage: 'Allows the translation of an entry into different languages',
```

Comment (gu-stav) [reply packages/plugins/i18n/admin/src/index.js:105]: I won't even start arguing with you 😂 Does your translation feel consistent with the one from D&P?

Comment (maevalienard) [reply packages/plugins/i18n/admin/src/index.js:105]: We are never too picky when it comes to copy. 

"languages" is fine to me. I'd keep "translating" however mostly for consistency with D/P :D

**Review by jhoward1994 — APPROVED** [review]

**Pull request merged**